js/join.js

```diff
@@ -126,32 +384,36 @@ function showPage(currentFlowState) {
 //Validators
 function arePersonalDetailsValid() {
   return (
-    state.value.trim().length > 3 &&
-    city.value.trim().length > 3 &&
-    country.value.trim().length > 3
+    state.value.trim().length >= 3 &&
+    city.value.trim().length >= 3 &&
+    country.value.trim().length >= 3
   );
 }
 
 function introPageValidator() {
   return (
-    introduction.value.trim().split(' ').length > 100 &&
-    skills.value.trim().split(' ').length > 6 &&
-    college.value.trim().split(' ').length > 5 &&
-    forFun.value.trim().split(' ').length > 100 &&
-    funFact.value.trim().split(' ').length > 100
+    introduction.value.trim().split(' ').length >= 100 &&
+    skills.value.trim().split(' ').length >= 5 &&
+    college.value.trim().split(' ').length >= 5 &&
+    forFun.value.trim().split(' ').length >= 100 &&
+    funFact.value.trim().split(' ').length >= 100
   );
 }
 
 function whyRdsPageValidator() {
-  return whyRds.value.trim().split(' ').length > 100 && foundFrom.value != '';
+  return whyRds.value.trim().split(' ').length >= 100 && foundFrom.value != '';
 }
 
 function dataValidator(element, size) {
-  if (element.value.trim().split(' ').length > size) {
-    element.classList.add('correct-data');
+  let counter = document.getElementById(element.id + 'Counter');
+  let words_left = size - element.value.trim().split(' ').length;
+  counter.innerText = `${words_left} word(s) left for your data to be valid`;
```

Comment:
This error message looks too verbose
A better message would be : 5 words left [This will convey how many more words the user has to write]

Reply:
but they can write more than 5 words.

Reply:
Still means the same

Reply:
"{y} more words required" means that the user needs to write y more words but they are not constrained to just y words. They just have to write atleast y more words

Reply:
Done